frontends/p4/fromv1.0/programStructure.cpp

```diff
@@ -1565,8 +1599,15 @@ ProgramStructure::convertControl(const IR::V1Control* control, cstring newName)
                 ::error("Cannot locate table %1%", c.first->table.name);
                 return nullptr;
             }
-            if (std::find(usedTables.begin(), usedTables.end(), tbl) != usedTables.end())
-                directCounters.emplace(c.first->table.name, c.first->name);
+            if (std::find(usedTables.begin(), usedTables.end(), tbl) != usedTables.end()) {
```

Comment:
What if a counter is used in multiple tables?
What if its count method is invoked multiple times?